pkgs/dash_analytics/lib/src/config_handler.dart

```diff
@@ -19,9 +19,9 @@ const String telemetryFlagPattern = r'^reporting=([0|1]) *$';
 /// from the configuration file
 ///
 /// Example:
-/// flutter-tools=2022-10-26,1
+/// flutter_tools=2022-10-26,1
 const String toolPattern =
-    r'^([A-Za-z0-9]+-*[A-Za-z0-9]*)=([0-9]{4}-[0-9]{2}-[0-9]{2}),([0-9]+)$';
+    r'^([A-Za-z0-9]+_*[A-Za-z0-9]*)=([0-9]{4}-[0-9]{2}-[0-9]{2}),([0-9]+)$';
```

Comment:
```suggestion
    r'^([A-Za-z0-9]+[A-Za-z0-9_]*)=([0-9]{4}-[0-9]{2}-[0-9]{2}),([0-9]+)$';
```

Or to simplify (if we don't mind allowing a leading underscore):

```dart
    r'^(\w+)=(\d{4}-\d{2}-\d{2}),(\d+)$';
```